sources/tools/Stride.TextureConverter/Backend/Wrappers/FINetWrapper/FreeImageStaticImports.cs

```diff
@@ -192,14 +192,14 @@ internal static partial class FreeImage
 		/// </summary>
 		[DllImport(FreeImageLibrary, EntryPoint = "FreeImage_DeInitialise")]
 		private static extern void DeInitialise();
-
+		
 		/// <summary>
 		/// Returns a string containing the current version of the library.
 		/// </summary>
 		/// <returns>The current version of the library.</returns>
 		public static unsafe string GetVersion() { return PtrToStr(GetVersion_()); }
 		[DllImport(FreeImageLibrary, CharSet = CharSet.Ansi, EntryPoint = "FreeImage_GetVersion")]
-		private static unsafe extern byte* GetVersion_();
+		public static unsafe extern byte* GetVersion_();
```

Comment:
Is it necessary?

Reply:
I do not remember why I changed the method to public, but it can be undone too (I sorted now the keywords correctly).